packages/next-swc/crates/next-core/src/lib.rs

```diff
@@ -48,7 +48,8 @@ pub use next_edge::context::{
     get_edge_chunking_context, get_edge_compile_time_info, get_edge_resolve_options_context,
 };
 pub use page_loader::{create_page_loader_entry_module, PageLoaderAsset};
-pub use turbopack_binding::{turbopack::node::source_map, *};
+use turbopack_binding::turbo;
+pub use turbopack_binding::{turbopack, turbopack::node::source_map};
```

Comment:
Where is this used?
Can we get rid of it?

Reply:
it is being imported by somewhere else, if I recall correctly next_build and another. 

looks like using reexport from turbopack_binding is not great, I'll look further.